medusa/restore_node.py

```diff
@@ -168,14 +168,18 @@ def invoke_sstableloader(config, download_dir, keep_auth, fqtns_to_restore, stor
                     logging.debug('Restoring table {} with sstableloader...'.format(table))
                     cql_username = 'foo' if config.cassandra.cql_username is None else config.cassandra.cql_username
                     cql_password = 'foo' if config.cassandra.cql_password is None else config.cassandra.cql_password
-                    output = subprocess.check_output([config.cassandra.sstableloader_bin,
-                                                      '-d', socket.getfqdn() if cassandra_is_ccm == 0
-                                                      else '127.0.0.1',
-                                                      '--storage-port', storage_port,
-                                                      '-u', cql_username,
-                                                      '--password', cql_password,
-                                                      '--no-progress',
-                                                      os.path.join(ks_path, table)])
+                    sstableloader_args = [config.cassandra.sstableloader_bin,
+                                          '-d', socket.getfqdn() if cassandra_is_ccm == 0
+                                          else '127.0.0.1',
+                                          '--username', cql_username,
+                                          '--password', cql_password,
+                                          '--no-progress',
+                                          '--storage-port', storage_port,
+                                          os.path.join(ks_path, table)]
+                    if storage_port == "7000":
+                        sstableloader_args.remove("--storage-port")
+                        sstableloader_args.remove(storage_port)
+                    output = subprocess.check_output(sstableloader_args)
```

Comment:
If I may object, I'd rather do something along the lines of:

```
                    sstableloader_args = [config.cassandra.sstableloader_bin,
                                          '-d', socket.getfqdn() if cassandra_is_ccm == 0
                                          else '127.0.0.1',
                                          '--username', cql_username,
                                          '--password', cql_password,
                                          '--no-progress']
                    if str(storage_port) != "7000":
                        sstableloader_args.append("--storage-port")
                        sstableloader_args.append(storage_port)
                    sstableloader_args.append(os.path.join(ks_path, table))
```

wdyt?
Also could you then squash the commits into a single one?
Other than that, LGTM 👍

Reply:
Makes sense. The list is smaller and there's no need to search in array and remove the positions like the way i did. I will change it and squash the commits.